crates/client/src/test_generator.rs

```diff
@@ -133,16 +133,17 @@ impl TestGenerator {
 
     /// Creates a new file with a given content on the specified `path` and `name`
     // todo: the function should be located in the different module, File module for example
+    #[allow(clippy::ptr_arg)]
```

Comment:
Why do we need this ignores? I understood [ptr_arg](https://insanitybit.github.io/gsblookup-rs/clippy/ptr_arg/index.html) recommends using slices instead of `String` and `Vector` refs in function arguments, but not one of the 3 functions use such arguments, right?

Reply:
Yep, you are 100% right, this is my mistake. I wanted to rewrite it using Path instead of PathBuf and I was having problems with lifetimes